frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/k8s-resource.ts

```diff
@@ -16,7 +16,7 @@ type BaseOptions = {
 type AdapterFunc = <D extends BaseOptions>(
   func: Function,
   knownArgs: string[],
-) => (options: D) => Promise<Response>;
+) => (options: D) => Promise<any>;
```

Comment:
This type was weird... the way we currently have our types makes it hard to type this one. I could have cleaned up our `AdapterFunc` usages and made this use Generics, but it seemed to be a lot of changes that were not needed.

This type is overridden by all users of this function so I think we are okay with this `any` for now.